docs/developers-guide/driver-changelog.md

```diff
@@ -26,16 +26,16 @@ title: Driver interface changelog
   `metabase.driver.sql.query-processor/honey-sql-version` is now deprecated and no longer called. All drivers are
   assumed to use Honey SQL 2.
 
-- The method `metabase.driver.sql-jdbc.sync.interface/active-tables` that we added in 47 has been updated to require
-  an additional argument: `database`.
-  The new function arglist is `[driver database connection schema-inclusion-filters schema-exclusion-filters]`.
-
 - The method `metabase.driver.sql.parameters.substitution/align-temporal-unit-with-param-type` is now deprecated.
   Use `metabase.driver.sql.parameters.substitution/align-temporal-unit-with-param-type-and-value` instead,
   which has access to `value` and therefore provides more flexibility for choosing the right conversion unit.
 
 ## Metabase 0.48.0
 
+- The multimethod `metabase.driver.sql-jdbc.sync.interface/current-user-table-privileges` has been added. All JDBC-
+  based drivers should implement this instead of the corresponding `metabase.driver` multimethod if they want to
+  leverage the built-in JDBC implementation of `metabase.driver/describe-database`.
+
```

Comment:
This doesn't need to be backported, so this will be in 49.

Reply:
My thinking was as follows:

We introduced the `metabase.driver` in 48, but after this change there's a difference in how SQL drivers should be implemented. I wanted to backport it so that the same driver implementation will work across both of the latest releases.

For now I'll just skip the backport, but let me know why you disagree with this reasoning. Is it because 3rd party SQL drivers are rare, this feature is still obscure, or because we don't need to worry about this kind of cross release compatibility for a given driver version?

Reply:
> Is it because 3rd party SQL drivers are rare, this feature is still obscure, or because we don't need to worry about this kind of cross release compatibility for a given driver version?

All of the above. We backported it without changing the changelog because this feature is still obscure there's good reason to assume no-one implemented the table-privileges feature unless they were crazy. We probably shouldn't have but that ship has sailed. And we don't need to bother backporting it because I suspect there will be conflicts and there's just not much of a reason to bother. Plus I find it confusing to add things to the backlog for a minor release, because there shouldn't be any changes to it after a major release under normal circumstances.